resources/assets/less/bem/search-entry.less

```diff
@@ -50,6 +50,18 @@
     opacity: 0;
   }
 
+  &--inner {
+    margin: 5px 0;
+  }
+
+  &__highlight {
+    em {
```

Comment:
elasticsearch gives the options of returning either `<em>`, `<em class="hlt{matchnumber}">` or some custom tag